From 8374e8059cc906292bc387759f7c72cc714e8e8f Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Fri, 23 Aug 2024 19:56:40 +0500
Subject: [PATCH] proprietary: Nuke dup AndroidAuto

---
 common/Android.bp | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/common/Android.bp b/common/Android.bp
index 86fe40f..a3d2fb2 100644
--- a/common/Android.bp
+++ b/common/Android.bp
@@ -156,19 +156,6 @@ android_app_import {
     product_specific: true,
 }
 
-android_app_import {
-    name: "AndroidAutoStubPrebuilt",
-    owner: "gapps",
-    apk: "proprietary/product/priv-app/AndroidAutoStubPrebuilt/AndroidAutoStubPrebuilt.apk",
-    presigned: true,
-    preprocessed: true,
-    dex_preopt: {
-        enabled: false,
-    },
-    privileged: true,
-    product_specific: true,
-}
-
 android_app_import {
     name: "AndroidPlatformServices",
     owner: "gapps",
-- 
2.34.1

