From 9732c9aec97231aec4bb02e322db550c53fb7983 Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Thu, 29 Aug 2024 05:36:56 +0000
Subject: [PATCH] libstagefright: Fix eac-joc decoder

---
 media/libstagefright/omx/SoftOMXPlugin.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/libstagefright/omx/SoftOMXPlugin.cpp b/media/libstagefright/omx/SoftOMXPlugin.cpp
index 636e02a..9418af5 100644
--- a/media/libstagefright/omx/SoftOMXPlugin.cpp
+++ b/media/libstagefright/omx/SoftOMXPlugin.cpp
@@ -60,6 +60,7 @@ static const kComponent kComponents[] = {
     { "OMX.google.gsm.decoder", "gsmdec", "audio_decoder.gsm" },
     { "OMX.dolby.ac3.decoder", "ddpdec", "audio_decoder.ac3" },
     { "OMX.dolby.eac3-joc.decoder", "ddpdec", "audio_decoder.eac3_joc" },
+    { "OMX.dolby.eac3_joc.decoder", "ddpdec", "audio_decoder.eac3_joc" },
     { "OMX.dolby.eac3.decoder", "ddpdec", "audio_decoder.eac3" },
 };
 
-- 
2.43.0

