From 96d65332b2e4bf892880a2e30f3689c0f95f02c9 Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Fri, 23 Aug 2024 19:57:58 +0500
Subject: [PATCH] packages: Point to other tracked gapps

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index c7679c7..30369d7 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -161,7 +161,7 @@ PRODUCT_VENDOR_PROPERTIES += \
     ro.egl.blobcache.multifile_limit=33554432
 
 ifeq ($(TARGET_BUILD_GAPPS),true)
-    $(call inherit-product-if-exists, vendor/google/gms/config.mk)
+    $(call inherit-product-if-exists, vendor/gapps/arm64/arm64-vendor.mk)
 else
     $(call inherit-product, external/svox/svox_tts.mk)
     $(call inherit-product, vendor/microg/products/gms.mk)
-- 
2.34.1

